server/channels/app/support_packet.go

```diff
@@ -173,6 +176,7 @@ func (a *App) generateSupportPacketYaml(c *request.Context) (*model.FileData, er
 		/* License */
 		LicenseTo:             licenseTo,
 		LicenseSupportedUsers: supportedUsers,
+		LicenseIsTrial:        strconv.FormatBool(isTrial),
```

Comment:
Can you please elaborate on the use case of having the trial license information is? Do we already know from sales force if a customer is using a trial license?

Reply:
Resolved via https://community.mattermost.com/private-core/pl/bqznibnfd7fhjmben8chkmprnr